komodo/build.py

```diff
@@ -198,7 +200,7 @@ def resolve(x):
 
     fakeroot = args.prefix if args.fakeroot else '.'
     args = parser.parse_args()
-    make(args.pkgfile, args.repofile, prefix = args.prefix,
+    make(args.pkgfile, args.repofile, prefix   = args.prefix,
```

Comment:
Should we revert this one also?

Reply:
yes, i just didn't notice that one ... now it's gone :+1: